simple/schema.py

```diff
@@ -97,6 +98,16 @@ class Gravity(enum.Enum):
     fldg = 'fld-g'
 
 
+class Parameter(enum.Enum):
+    """Enumeration for derived/inferred parameters"""
+    mass = 'mass'
+    radius = 'radius'
+    log_g = 'log_g'
+    T_eff = 'T_eff'
+    metallicity = 'metallicity'  # Z
+    CO = 'CO'  # C/O
```

Comment:
maybe we should be more pedantic here. I think `CO` is too ambiguous/misleading. Maybe `C/O ratio`? I dont think there's a problem with the slash in the string is there? If so, `C-to-O ratio`?

Reply:
I need to confirm (and maybe write a test for) but I'm 80% sure that what gets stored in the database is the variable, not the string. Which means things like `C/O` and `C-to-O` are invalid.

Reply:
C_to_O_ratio

Reply:
Switching to an actual table, so the enumeration question is no longer relevant